src/selectors/url-state.js

```diff
@@ -341,3 +342,53 @@ export const getCommittedRangeLabels: Selector<string[]> = createSelector(
   getAllCommittedRanges,
   CommittedRanges.getCommittedRangeLabels
 );
+
+function _shouldAllowSymbolServerUrl(symbolServerUrl) {
+  try {
+    const localhostHostnames = ['localhost', '127.0.0.1', '0.0.0.0'];
```

Comment:
I'm a bit surprised by the use of `0.0.0.0`, my understanding is that this is a routing address (meaning "everything") and not a target address...  But I see that this works locally, I'm a bit puzzled because I see no reference to this use. Do you have some pointers?

Reply:
Good question! I hadn't thought about it before, I just put it here because local webservers like [simple-http-server](https://github.com/TheWaWaR/simple-http-server) output a URL like http://0.0.0.0:8000 to the console when run, so I've just been using that for testing.

I found a stackoverflow answer with an explanation at https://serverfault.com/a/78058 :
> 0.0.0.0, in this context, means "all IP addresses on the local machine" (in fact probably, "all IPv4 addresses on the local machine"). So, if your webserver machine has two IP addresses, 192.168.1.1 and 10.1.2.1, and you allow a webserver daemon like apache to listen on 0.0.0.0, it will be reachable at both of those IP addresses. But only to what can contact those IP addresses and the web port(s).

Reply:
And there's also this answer: https://stackoverflow.com/a/55646032/6740906
> However, for practical reason, many client software treat 0.0.0.0 as localhost. Such software list includes: Chrome, Firefox, Safari, curl, telnet etc. Because, as many web server software launch with messages like "listen on 0.0.0.0...", allow visiting to 0.0.0.0 is very user-friendly for junior developers.

Reply:
I'm going to remove 0.0.0.0 from the list.

Reply:
ok, thanks for the pointers!

FTR If that makes things easier in the future, I'm fine with adding it back :-)

Reply:
Well, first I was going to keep it, but then I added a comment explaining it, and the comment ended up being around 5 lines, and the core message was "this shouldn't work but it does because browsers have chosen to make it work anyway in spite of the spec", so I determined that it wasn't worth putting in.